examples/hello_world/lib/arabic.dart

```diff
@@ -2,6 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
 
-void main() => runApp(const Center(child: Text('برنامج أهلا بالعالم', textDirection: TextDirection.rtl)));
```

Comment:
This example is designed to be as minimal as possible to show "hello world". It should do so using only the widgets layer and nothing else. I.e. it shouldn't depend on Material.

Reply:
Ok  I'm going to undo